### ISSUE TITLE
question: query for cards that are from standard or expanded legal

### ISSUE THREAD

**Issue by dmtrs** [tcgdex/cards-database#532]
### Where are you having the issue

- [X] GraphQL
- [ ] JSON API

### Is the issue affecting one of the two elements below ?

- [ ] Translations
- [X] Definitions Files (Open API, GraphQL Schema, etc)

### Description

How can we should the `CardFilters` be extended to support search for standard (`true`) cards?

From what I understand from the code the `CardFilters` should work (as definition) with nested data? 

Should there be a "transformation" to query from CardFilters object?

**Comment by Aviortheking** [comment]
Nope, I seem to have forgotten to update the CardFilter object with the childs of the card possible values, I'll try to do it this week !
